susemanager-utils/susemanager-sls/src/modules/kiwi_info.py

```diff
@@ -6,6 +6,13 @@
 
 log = logging.getLogger(__name__)
 
+# Kiwi version is always in format <MAJOR:MINOR:PATCH> with numeric values
+KIWI_VERSION_REGEX=r'\d+\.\d+\.\d+'
```

Comment:
To me it looks like the comment and the regex aren't aligned, `r'\d+\.\d+\.\d+'` doesn't match `MAJOR:MINOR:PATCH`
```python
>>> re.match(KIWI_VERSION_REGEX, "1:2:3")
>>> re.match(KIWI_VERSION_REGEX, "1.2.3")
<re.Match object; span=(0, 5), match='1.2.3'>
```
Maybe it would be easier to use named groups here as well and get rid of the comment?